test/integration/error_spam_test.go

```diff
@@ -161,6 +169,18 @@ func TestErrorSpam(t *testing.T) {
 			}
 			cleanupLogFiles(t, logFiles)
 
+			singleRun, err := Run(t, exec.CommandContext(ctx, Target(), args...))
```

Comment:
Curious what the reasoning is for doing a single run and then doing the rest after. If there any issues with just doing the full looped run below?

Reply:
Not a blocker but it will return the error faster in case logs are not getting generated for a single case instead of the command running 175 times and then giving the error. Removed the changes from the branch. Can take it up as a enhancement if needed